bin/riemann-rabbitmq

```diff
@@ -17,6 +17,7 @@ class Riemann::Tools::Rabbitmq
   opt :monitor_pass, 'RabbitMQ monitoring user password', type: :string
   opt :monitor_port, 'RabbitMQ monitoring port', type: :int, default: 15672
   opt :monitor_host, 'RabbitMQ monitoring host', type: :string, default: "localhost"
+  opt :monitor_ssl, 'RabbitMQ ssl', type: :string, default: "false"
```

Comment:
I might actually call this `:monitor_tls` since it's TLS, not SSL, right? Or maybe have it be a string but call it `:protocol`?, and have values be `http` or `https`?